xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/viewers/tags.css

```diff
@@ -7,6 +7,9 @@
     and the next footer item (e.g. the document creator). */
   padding: 3px 2em 3px 0;
 }
+.tag-visibility {
```

Comment:
you don't need this class, as I mentioned there's a dedicated CSS class named `hidden` in xwiki for this.

Reply:
Hi, please check again